### DISCUSSION TITLE
[Announcement] Adjustment of the evaluation criteria for HW4 (++ One more thing)

### DISCUSSION THREAD

**Discussion by RiceBiscuits** [prosyslab-classroom/cs348-information-security#445] [Announcements]
Hello, Students.

An evaluation criteria for HW4 have been adjusted.
The changed criteria is the coverage of the json analyzer.

Before: 
coverage lines(line, column) of the json parser is greater than **500** -> 20pts

After:
coverage lines(line, column) of the json parser is greater than **800** -> 20pts

+++++ ADDED +++++++
In this homework, we will not push a json parser used on the evaluation to student's repositories.
++++++++++++++++++

Thank you.

**Comment by Junseon00** [comment]
You can check the reference in here. 
https://docs.google.com/document/d/1zIM83xkeyy8lBDJa1C6QVWM2rqUFxoyX9FIHqbQTSCg/edit?usp=sharing

**Comment by RiceBiscuits** [comment]
Please check this announcement again. One more Hw4 evaluation notice was added.

+++++ ADDED +++++++
In this homework, we will not push a json parser used on the evaluation to student's repositories.
++++++++++++++++++
